File: p12577.py
count=0
while (1):
	try:
		line= input()
	except EOFError:
		break 
	count=count+1     
	if line[0]=='*':
		break
	elif line[0]=='H':
		print("Case ",count,': Hajj-e-Akbar')
	else:
		print('Case '+count+': Hajj-e-Asghar')

File:  p12577.cpp
#include <iostream>
using namespace std;

int main()
{
    string line;
    int count=1;

    while(cin>>line)
    {
        if(line[0]=='*')
        break;
        else if(line[0]=='H')
            cout<<"Case "<<count++<<": Hajj-e-Akbar"<<endl;
        else
            cout<<"Case "<<count++<<": Hajj-e-Asghar"<<endl;
    }
}
